### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Screen Reader: While navigating to the “Save as CSV” link NVDA is reading “Virtual Document Document Document”

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/vscode-mssql#1228]
**Environment Details:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.18362 Build 18362)
Screen reader: NVDA, JAWS

**MAS Violated:** MAS 1.3.1

**Repro Steps:**
1.       Open Visual Studio Code in your system
2.       Navigate to "Extensions"   and install "SQl Server (mssql)" extension
3.       Press F1 and select MS SqL : Connect and connect to any sql server
4.       Write code (SELECT * FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query""   
5.       Start NVDA   
6.       Tab till “Save to CSV” link    
7.       Verify if NVDA is reading “Save to CSV” link properly

**Actual Result:** 
While navigating to the “Save as CSV” link NVDA is reading “Virtual Document Document Document Angular 2 Quick start Document” which is not present on the screen

**Expected Result:** 
While navigating to the “Save as CSV” link NVDA should read as “Save as CSV” link but should not read the information which are not present on the screen

**User Impact:** 
Screen Reader users will find difficulty in understanding if Screen reader is not reading the link properly

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_1228_NVDA reading documet many times](https://user-images.githubusercontent.com/41109135/56639584-a4ee9880-668e-11e9-885d-486a795406e9.png)

**Comment by alanrenmsft** [comment]
this has been fixed and is working properly in the latest version.

**Comment by Kothoor** [comment]
BM_MSSQL_MAY2020

**Comment by fsteffi** [comment]
#closed;
fixed on
Version: 1.79.0-insider (user setup)
Commit: fa8eefda18bf099d82b58453f8fe5d3010196fa0
Date: 2023-05-31T03:06:45.096Z
Electron: 22.5.4
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25370
![image](https://github.com/microsoft/vscode-mssql/assets/24580570/ea373c97-21e9-4dfe-ad9e-aeb5dcdf5137)

**Comment by InduPriya1805** [comment]
GitHubTags:#BM_MSSQL_MAY2020;
